src/mlpack/methods/reinforcement_learning/replay/random_replay.hpp

```diff
@@ -0,0 +1,142 @@
+/**
+ * @file random_replay.hpp
```

Comment:
Yeah there is also prioritized replay but I'm not going to implement it in the near future. But I sill think a separate folder is better because once we add a new replay method, a separate folder can provide backward compatibility automatically. Otherwise user may have to modify their `#include` clause.